Api/Validators/Event/UpdateEventRequestValidator.cs

```diff
@@ -13,6 +13,7 @@ public UpdateEventRequestValidator()
     {
         RuleFor(x => x.MustJoinUntil)
             .LessThan(x => x.Time)
+            .DateTimeInFuture()
```

Comment:
To trzeba będzie walidować w samej, końcówce, sprawdzając, że została zmieniona. Bo jeśli użytkownik będzie chciał zaktualizować inne info o evencie, to i tak będzie musiał podać tą datę. A jeśli ona już minęła, to dostanie błąd, mimo to, że jej nie zmieniał.

Reply:
Przyjrzę się